### PR TITLE
Automated cherry pick of 1775 release 7.0

### PR DESCRIPTION
**Pull request by adityadani**

### CODE REVIEW

File: SDK_CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Changelog
 
 ## Releases
+### v0.69.29 - (11/05/2020)
```
Comment (lpabon) [inline SDK_CHANGELOG.md:4]: This needs to be bumped

File: api/server/sdk/job.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ func (j *JobServer) GetJobStatus(
 	return j.cluster().GetJobStatus(ctx, req)
 }
 
-func (j *JobServer) EnumerateJobs(
+func (j *JobServer) Enumerate(
```
Comment (lpabon) [inline api/server/sdk/job.go:43]: SDK Version needs to be bumped.

**Review by lpabon — CHANGES_REQUESTED** [review]

File: SDK_CHANGELOG.md (hunk 1 above)
Comment (adityadani) [reply SDK_CHANGELOG.md]: Done

File: api/server/sdk/job.go (hunk 2 above)
Comment (adityadani) [reply api/server/sdk/job.go:43]: Done

**Pull request merged**